parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -333,6 +333,10 @@ if $REBOOTREQUIRED; then
         # logs_to_events should not be run on & commands
         aptmarkWALinuxAgent unhold &
     fi
+    # Start the mig-partition.service for Mariner after rebooting to complete the rest ofd the cxonfigurations
+    if [[ $OS == $MARINER_OS_NAME ]]; then
+        systemctlEnableAndStart mig-partition || exit $ERR_SYSTEMCTL_START_FAIL
```

Comment:
isn't this already required elsewhere for both distros?

Reply:
This code has already been eliminated.